### PR TITLE
Chat to Secure Conversation transfer

### PR DESCRIPTION
**Pull request by EgorovEI**
MOB-3630

**What was solved?**
This PR introduces handling Live chat to Secure Conversations transferring

**Release notes:**

 - [x] Feature
 - [ ] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

- [x] Is the feature sufficiently tested? All tests fixed? Necessary unit, acceptance, snapshots added? Check that at least new public classes & methods are covered with unit tests
 - [ ] Did you add logging beneficial for troubleshooting of customer issues?
 - [ ] **Did you add new logging?** We would like the logging between platforms to be similar. Refer to **Logging from iOS SDKs** → **Things to consider for newly added logs** in Confluence for more information.

### CODE REVIEW

**Comment by EgorovEI** [comment]
!squash

File: GliaWidgets/Sources/Interactor/Interactor.swift [hunk 1]
```diff
@@ -289,7 +290,13 @@ extension Interactor: CoreSdkClient.Interactable {
     }
 
     var onLiveToSecureConversationsEngagementTransferring: CoreSdkClient.EngagementTransferringBlock {
-        return {}
+        return { [weak self, environment] in
+            environment.log.prefixed(Self.self).info(
```
Comment (igorkravchenko) [inline GliaWidgets/Sources/Interactor/Interactor.swift:294]: Just want to confirm if this newly introduced log is added to logging table as suggested in [guide](https://glia.atlassian.net/wiki/x/awD31Q).

**Review by igorkravchenko — APPROVED** [review]
Looks good. Left one question regarding added log.

File: GliaWidgets/Sources/Interactor/Interactor.swift (hunk 1 above)
Comment (EgorovEI) [reply GliaWidgets/Sources/Interactor/Interactor.swift:294]: This log is added to the Google sheet

**Pull request merged**